scipy/misc/__init__.py

```diff
@@ -32,13 +28,25 @@
    imshow - Simple showing of an image through an external viewer [requires Pillow]
    info - Get help information for a function, class, or module
    lena - Get classic image processing example image Lena
+   toimage - Takes a numpy array and returns a PIL image [requires Pillow]
+   source - Print function source code
```

Comment:
Ah yes, misread - sorry about that. While you're at it, I think deprecating ``who`` and ``source`` definitely makes sense. They're just numpy aliases. 

``info`` has a slight modification, but I'm not sure it's of any use. It certainly doesn't belong anywhere other than ``misc``, so if we're getting rid of that.... and doesn't add much over ``help`` or if needed ``np.info(..., toplevel='scipy')``.